code/src/java/pcgen/system/PCGenSettings.java

```diff
@@ -104,6 +106,8 @@ private PCGenSettings()
 		setProperty(VENDOR_DATA_DIR, "@vendordata");
 		setProperty(HOMEBREW_DATA_DIR, "@homebrewdata");
 		setProperty(CUSTOM_DATA_DIR, "@data/customsources".replace('/', File.separatorChar));
+		OutputDB.registerBooleanPreference(
```

Comment:
Thanks Tom, that makes sense to me now. The initialization as written should be fine then.